docs/how_tos/running_one_off_scripts.rst

```diff
@@ -0,0 +1,14 @@
+===================================
+How to use the scripts in /scripts
+===================================
+
+First, read docs/decisions/0005-store-one-off-scripts.rst to understand why scripts of this nature are stored in this repository.
+
+To apply the changes from a script to your repository
+-----------------
+
+1. (for edx.org): Run the cleanup-python-code job in Jenkins with the contents of the script (minus the #!) as the input.
```

Comment:
Will all the scripts in /scripts directory always use cleanup-python-code? Can you change this to be specific to just update_setup_py.sh?

Reply:
I was thinking they should always be compatible with cleanup python code since that is our main tool for making these kinds of changes across repositories.